openach/models.py

```diff
@@ -197,12 +199,23 @@ class EvidenceSource(models.Model):
         max_length=URL_MAX_LENGTH,
         help_text=_('A source (e.g., news article or press release) corroborating the evidence'),
     )
+    source_url_title = models.CharField(
```

Comment:
- Can we change these fields to source_title and source_description, respectively? The fact that we're fetching them automatically from the URL is just a detail
- Quotes should be single-quotes
- Indentation and spacing needs to be fixed